platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdminTest.java

```diff
@@ -208,6 +183,31 @@ public void testWriteSystemProperties() throws Exception {
     }
   }
 
+  @Test
+  public void testWritePlaceholderProperties() throws Exception {
```

Comment:
❓ Do these unit test confirm that the properties are written back to the custom.system.properties file? Is verifying `SystemPropertiesAdmin#readSystemProperties` enough to verify that custom.system.properties was updated as expected?